packages/services/cdn-worker/src/artifact-handler.ts

```diff
@@ -1,27 +1,24 @@
-import itty from 'itty-router';
+import { createRouter, Response, RouterRequest } from 'fets';
 import zod from 'zod';
-import { createFetch, type Request } from '@whatwg-node/fetch';
 import { type Analytics, createAnalytics } from './analytics';
 import { type ArtifactsType } from './artifact-storage-reader';
-import { InvalidAuthKeyResponse, MissingAuthKeyResponse } from './errors';
+import { createInvalidAuthKeyResponse, createMissingAuthKeyResponse } from './errors';
 import type { KeyValidator } from './key-validation';
 
-const { Response } = createFetch({ useNodeFetch: true });
-
 type ArtifactRequestHandler = {
   getArtifactAction: (
     targetId: string,
     artifactType: ArtifactsType,
-    eTag: string | null,
+    eTag?: string | null,
   ) => Promise<
     { type: 'notModified' } | { type: 'notFound' } | { type: 'redirect'; location: string }
   >;
   isKeyValid: KeyValidator;
   analytics?: Analytics;
   fallback?: (
-    request: Request,
+    request: Request | RouterRequest,
     params: { targetId: string; artifactType: string },
-  ) => Promise<Response | undefined>;
+  ) => Promise<any>;
```

Comment:
why any?

Reply:
Somehow default `Response` and `TypedResponse` were conflicting. Adding a workaround to make it pass for now.